controllers/networking-calico/example/20-network.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: extensions.gardener.cloud/v1alpha1
+kind: Network
+metadata:
+  name: calico-network
+  namespace: default
+spec:
+  type: calico
+  cloudProvider: aws
+  clusterCIDR: 192.168.0.0/24
```

Comment:
clusterCIDR is the name of flag in the Cloud-controller manager, this is important to relate because the CIDR defined here should also match the cluster CIDR in the CCM.

Reply:
Fine, but in our API we have other conventions and we should not start mixing it.